TurkishFilter/sections/specific.txt

```diff
@@ -589,7 +589,7 @@ haberinkalemi.com##div[class*="ads"]
 bestdizi.com##a[href="https://panelsmm.net/"]
 bestdizi.com##iframe[src^="https://assets.cdnxz.com/"]
 forumatmosfer.com###wrapperindex > div[style*="text-align: center;"]:not([style*="background: #666666;"])
-||i.ibb.co/pf6kcmz/Banner-header-*.gif$domain=artigercek.com
+||i.ibb.co/K6PYZF9/Banner-header-*.gif$domain=artigercek.com
```

Comment:
1. Just `/Banner-header-*.gif$domain=artigercek.com`.
2. I'd block this element in AdGuard Annoyances - annoyances.txt -> `artigercek.com#?#.container > div.row:has(> div[class^="col-"] div.small + a.social-icon)`.
3.  Unblock their social links in AdGuard Social Media -> whitelist.txt.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/90575
artigercek.com#@#.social-icon
```

Reply:
@zloyden I don't know how to make changes in Pull request. Please add these rules yourself.

Reply:
Ok, I've done it.